src/gleam/string.gleam

```diff
@@ -208,11 +270,8 @@ pub fn append(to first: String, suffix second: String) -> String {
 /// concat(["never", "the", "less"]) == "nevertheless"
 /// ```
 ///
-pub fn concat(strings: List(String)) -> String {
-  strings
-  |> iodata.from_strings
-  |> iodata.to_string
-}
+pub external fn concat(strings: List(String)) -> String =
+  "unicode" "characters_to_binary"
```

Comment:
`iodata` not necessary?

Reply:
I guess it's not needed here if the tests still pass :)